### PR TITLE
Refactoring TechnologyBlipCfgControllerTests done

### PR DESCRIPTION
**Pull request by A9afon**
Refactoring TechnologyBlipCfgControllerTests done

### CODE REVIEW

File: src/test/java/com/a5lab/axion/domain/technology_blip/TechnologyBlipCfgControllerTests.java [hunk 1]
```diff
@@ -52,14 +52,40 @@ public void shouldGetTechnologyBlips() throws Exception {
...
-
-    final RingDto ringDto = new RingDto(10L, null, "My ring title", "My ring description", 0, "#fbdb84", true, null);
+    final SegmentDto segmentDto = new SegmentDto();
+    segmentDto.setId(10L);
+    segmentDto.setRadar(null);
+    segmentDto.setTitle("My segment title");
+    segmentDto.setDescription("My segment description");
+    segmentDto.setPosition(1);
+    segmentDto.setActive(true);
+    segmentDto.setTechnologyBlipList(null);
```
Comment (arufanov) [inline src/test/java/com/a5lab/axion/domain/technology_blip/TechnologyBlipCfgControllerTests.java:62]: @A9afon 
* should we use this setter?

File: src/test/java/com/a5lab/axion/domain/technology_blip/TechnologyBlipCfgControllerTests.java [hunk 2]
```diff
@@ -52,14 +52,40 @@ public void shouldGetTechnologyBlips() throws Exception {
...
+
+    final RingDto ringDto = new RingDto();
+    ringDto.setId(10L);
+    ringDto.setRadar(null);
+    ringDto.setTitle("My ring title");
+    ringDto.setDescription("My ring description");
+    ringDto.setPosition(0);
+    ringDto.setColor("#fbdb84");
+    ringDto.setActive(true);
+    ringDto.setTechnologyBlipList(null);
```
Comment (arufanov) [inline src/test/java/com/a5lab/axion/domain/technology_blip/TechnologyBlipCfgControllerTests.java:72]: @A9afon 
* should we use this setter?

**Review by arufanov — CHANGES_REQUESTED** [review]

File: src/test/java/com/a5lab/axion/domain/technology_blip/TechnologyBlipCfgControllerTests.java (hunk 1 above)
Comment (A9afon) [reply src/test/java/com/a5lab/axion/domain/technology_blip/TechnologyBlipCfgControllerTests.java]: Shouldn't, I'll fix it

File: src/test/java/com/a5lab/axion/domain/technology_blip/TechnologyBlipCfgControllerTests.java (hunk 2 above)
Comment (A9afon) [reply src/test/java/com/a5lab/axion/domain/technology_blip/TechnologyBlipCfgControllerTests.java]: Shouldn't, I'll fix it

**Review by arufanov — APPROVED** [review]
@A9afon 
* thanks!

**Pull request merged**